include/zephyr/bluetooth/l2cap.h

```diff
@@ -594,6 +594,14 @@ int bt_l2cap_chan_disconnect(struct bt_l2cap_chan *chan);
  *  case of an error the caller retains the ownership of the buffer.
  *
  *  @return 0 in case of success or negative value in case of error.
+ *  @return -EINVAL if `buf` or `chan` is NULL.
+ *  @return -EINVAL if `chan` is not either BR/EDR or LE credit-based.
+ *  @return -EINVAL if buffer doesn't have enough bytes reserved to fit header.
+ *  @return -ENOBUFS if L2CAP TX context couldn't be allocated. Retry later.
```

Comment:
Verify

Reply:
this has [now been removed](https://github.com/zephyrproject-rtos/zephyr/pull/67386)